plugins/servicenow/help.md

```diff
@@ -17,12 +17,6 @@ Note: This plugin affects only the underlying tables in a ServiceNow instance, n
 * ServiceNow username, password, client ID, and client secret (for OAuth authentication) 
 * ServiceNow instance name
 
-Please note that to use certain actions it's necessary to use scopes that have permissions on certain tables. Depending on the actions, it's necessary to add specific auth scopes:
```

Comment:
I'm only noticing this got removed from the help.md, should it be kept in?

Reply:
Yes, it's been stripped in the refresh. Let me add it back.

Reply:
https://github.com/rapid7/insightconnect-plugins/pull/2646